docs/services/gcp-services-guide.md

```diff
@@ -0,0 +1,536 @@
+```
+*Output*
+```
+NAME                 FIRST_CONDITION   SECOND_CONDITION
+example-connection   True              True
+```
+Wait for both conditions to be true to continue.
+
+## Install Wordpress
+Installing Wordpress requires creating a Kubernetes `Deployment` and load
+balancer `Service`. We will point the deployment to the `wordpressmysql` secret
+that we specified in our claim above for the Wordpress container environment
+variables. It should have been populated with our MySQL connection details after
+the claim became `Bound`.
+
+* Check to make sure `wordpressmysql` exists and is populated:
+
+*Command*
+```bash
+kubectl describe secret wordpressmysql -n app-project1-dev
+```
+
+*Output*
+```bash
+Name:         wordpressmysql
+Namespace:    app-project1-dev
+Labels:       <none>
+Annotations:  <none>
+
+Type:  Opaque
+
+Data
+====
+endpoint:  75 bytes
+password:  27 bytes
+username:  58 bytes
+```
+
+* Define the `Deployment` and `Service` in `wordpress-app.yaml`:
+```bash
+cat > wordpress.yaml <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: app-project1-dev
+  name: wordpress
+  labels:
+    app: wordpress
+spec:
+  selector:
+    matchLabels:
+      app: wordpress
+  template:
+    metadata:
+      labels:
+        app: wordpress
+    spec:
+      containers:
+        - name: wordpress
+          image: wordpress:4.6.1-apache
+          env:
+            - name: WORDPRESS_DB_HOST
+              valueFrom:
+                secretKeyRef:
+                  name: wordpressmysql
+                  key: endpoint
+            - name: WORDPRESS_DB_USER
+              valueFrom:
+                secretKeyRef:
+                  name: wordpressmysql
+                  key: username
+            - name: WORDPRESS_DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: wordpressmysql
+                  key: password
+          ports:
+            - containerPort: 80
+              name: wordpress
+---
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: app-project1-dev
+  name: wordpress
+  labels:
+    app: wordpress
+spec:
+  ports:
+    - port: 80
+  selector:
+    app: wordpress
+  type: LoadBalancer
+EOF
+
+kubectl apply -f wordpress.yaml
+```
+
+* You can verify creation with the following command and output:
+
+*Command*
+```bash
+kubectl get -f wordpress.yaml 
+```
+*Output*
+```bash
+NAME                        READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/wordpress   1/1     1            1           11m
+
+NAME                TYPE           CLUSTER-IP    EXTERNAL-IP   PORT(S)        AGE
+service/wordpress   LoadBalancer   10.0.128.30   52.168.69.6   80:32587/TCP   11m
+```
+
+If the `EXTERNAL-IP` field of the `LoadBalancer` is `<pending>`, wait until it
+becomes available, then navigate to the address. You should see the following:
+
+![alt azure wordpress](azure-wordpress.png)
```

Comment:
Is this right? I was not expecting a link to an `azure*` file from the GCP guide.

Reply:
This link will actually point to just a picture of the Wordpress starter page. We can use it for each of the services guide because it will look the same, but you are right that we should change the name. @muvaf I'll change image to `wordpress-start.png`.

Reply:
It's a generic Wordpress welcome page image that I borrowed from https://github.com/crossplaneio/crossplane/pull/800 , not specific to azure. It should work when that PR is merged.